playwright-e2e/pages/eligibility-pages.ts

```diff
@@ -0,0 +1,152 @@
+    ]);
+    await expect(this.moreAboutAdoptionLink).toBeVisible();
+    await this.continueButton.click();
+    await expect(this.mainContent).toContainText([
+      "Check you're eligible to adopt You cannot apply to adopt You cannot apply to adopt the child because they've been married or in a civil partnership. More about adoption",
+    ]);
+  }
+
+  async youAndApplicantUnder21(): Promise<void> {
+    await this.continueButton.click();
+    await this.applyMoreThanOneChildNo.check();
+    await this.continueButton.click();
+    await this.childUnder18Yes.check();
+    await this.continueButton.click();
+    await this.isChildMarriedOrCivilPartnershipNo.check();
+    await this.continueButton.click();
+    await this.areYouAndApplicantOver21No.check();
+    await expect(this.labelWarningYouCanOnly).toContainText([
+      'You must be 21 or over to adopt a child. This includes any other applicant.',
+    ]);
+    await expect(this.moreAboutAdoptionLink).toBeVisible();
+    await this.continueButton.click();
+    await expect(this.mainContent).toContainText([
+      "Check you're eligible to adopt You cannot apply to adopt You cannot apply to adopt the child until you, and your partner if applicable, are 21. More about adoption",
+    ]);
+  }
+
+  async notUKResident(): Promise<void> {
+    await this.continueButton.click();
+    await this.applyMoreThanOneChildNo.check();
+    await this.continueButton.click();
+    await this.childUnder18Yes.check();
+    await this.continueButton.click();
+    await this.isChildMarriedOrCivilPartnershipNo.check();
+    await this.continueButton.click();
+    await this.areYouAndApplicantOver21Yes.check();
+    await this.continueButton.click();
+    await this.uKCountryOfResidenceNo.check();
+    await expect(this.labelWarningYouCanOnly).toContainText([
+      'You cannot apply to adopt a child unless you have a permanent home here.',
+    ]);
+    await expect(this.moreAboutAdoptionLink).toBeVisible();
+    await this.continueButton.click();
+    await expect(this.mainContent).toContainText(['You cannot apply to adopt']);
+  }
+
+  async notUKResident12Months(): Promise<void> {
+    await this.continueButton.click();
+    await this.applyMoreThanOneChildNo.check();
+    await this.continueButton.click();
+    await this.childUnder18Yes.check();
+    await this.continueButton.click();
+    await this.isChildMarriedOrCivilPartnershipNo.check();
+    await this.continueButton.click();
+    await this.areYouAndApplicantOver21Yes.check();
+    await this.continueButton.click();
+    await this.uKCountryOfResidenceYes.check();
+    await this.continueButton.click();
+    await this.uKCountryOfResidence12MonthsYes.check();
+    await expect(this.labelWarningYouCanOnly).toContainText([
+      'You cannot apply to adopt a child unless you have a permanent home here.',
+    ]);
+    await this.continueButton.click();
+    await expect(this.mainContent).toContainText(['You cannot apply to adopt']);
+  }
+}
```

Comment:
These look great.  Were you intending to include scenarios for applyMoreThanOneChildYes in this PR?  

If not, let me know.  If would be a good test for the dynamic fee change that is coming up (the page refers to a fee of £183, but will change to £201 soon).

Reply:
yeah i think the plan is to build up the test suit little by little... 
I can start the next PR for 'mutiple child spec' now
this PR was mostly done by Georgina but I made a new PR as it was pointing to the out of date clusters - Georgina is on AL until Monday :)